.stickler.yml

```diff
@@ -0,0 +1,5 @@
+linters:
```

Comment:
I noticed you mentioned in a commit that it doesn't work wit hMicroverse recommended config. Are you sure?
See previously mentioned link and this link https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js for style config js file.

Reply:
Thanks Maya. I will try it again. Cheers.

Reply:
Yeah. It has worked now. I guess I rushed the first time. Thanks again!

Reply:
Yay! 🙂